gpcontrib/arenadata_toolkit/arenadata_toolkit--1.0.sql

```diff
@@ -129,3 +129,196 @@ BEGIN
 END$$
 LANGUAGE plpgsql VOLATILE
 EXECUTE ON MASTER;
+
+CREATE FUNCTION arenadata_toolkit.adb_get_relfilenodes(tablespace_oid OID)
```

Comment:
You shouldn't modify 1.0 script. How does a user update an existing installation by these changes?

Reply:
> How does a user update an existing installation by these changes?

Has the original version already been released?